### PR TITLE
Remove ejabberd

### PR DESCRIPTION
**Pull request by stvstnfrd**
We don't use this, nor have we, for some time.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
What should we do about the database? Should we run the migration backwards to delete the table? Doesn't this have its own DB entirely; should we nuke it too? Anything else to consider?

**Comment by stvstnfrd** [comment]
Some subset of @dcadams @kluo @caesar2164

**Comment by caesar2164** [comment]
HOLY CRAP! NOW THAT IS A NICE CHANGESET! -5702 lines, KILLER! :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by kluo** [comment]
+1

**Comment by stvstnfrd** [comment]
@dcadams Any thoughts on what to do w/ the database/migrations?

**Comment by stvstnfrd** [comment]
Per F2F, we'll run the migration backward and remove the table.

File: lms/djangoapps/jabber/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,39 +0,0 @@
-# NOTE: this is for *test purposes only*. We need some sort of DB set
-#       up for testing, but in dev/prod, whoever provisions ejabberd
-#       should provision its auth database separately, *not* using
-#       this migration data (as it's far from complete).
```
Comment (stvstnfrd) [inline lms/djangoapps/jabber/migrations/0001_initial.py]: This explains a bit more of what we were (or weren't) seeing earlier @dcadams .

**Comment by stvstnfrd** [comment]
I'm merging this now.

**Pull request merged**